publish.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+set -euo pipefail
+
+JOB_TYPE=${JOB_TYPE:-local}
```

Comment:
TODO (follow up if possible): document how to use this script in https://github.com/openshift/dynamic-plugin-sdk/blob/main/docs/publish-packages.md from a developers perspective (pull request and adding the `publish-sdk` label) and from a maintainer perspective (manual publish and using `FORCE_PUBLISH`).